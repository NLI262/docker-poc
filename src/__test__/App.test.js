// import React from 'react';
// import { render } from '@testing-library/react';
// import App from './App';

// test('renders learn react link', () => {
//   const { getByText } = render(<App />);
//   const linkElement = getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });


import React from 'react';
import { shallow } from 'enzyme';


import MainComponent from '../App.js';
describe('MyComponent', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<MainComponent />);

    expect(component).toMatchSnapshot();
  });
});
