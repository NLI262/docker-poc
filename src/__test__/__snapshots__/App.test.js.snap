// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly in "debug" mode 1`] = `
<div
  className="App"
>
  <h1
    className="responsive-header"
  >
    Responsive With
  </h1>
  <h1
    className="scss-container"
  >
    SCSS 
  </h1>
  <table
    className="table-element"
  >
    <tr>
      <th>
        Month
      </th>
      <th>
        Savings
      </th>
    </tr>
    <tr>
      <td>
        January
      </td>
      <td>
        $100
      </td>
    </tr>
    <tr>
      <td>
        February
      </td>
      <td>
        $501
      </td>
    </tr>
  </table>
</div>
`;
